foedag_version.h.in

```diff
@@ -3,4 +3,5 @@
 #define FOEDAG_VERSION      @VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@ 
```

Comment:
It is make sense to rename this to FOEDAG_BUILD since it suppose to represent build but not version

Reply:
Fixed this.